datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -416,6 +416,50 @@ from data
 ----
 1
 
+# csv_query_distance
+query R
+SELECT distance(c2, c12) FROM aggregate_test_100
+----
+27.565541154252
+
+# single_row_query_distance
+query R
+select distance(sq.column1, sq.column2) from (values (1.1, 2.2)) as sq
+----
+1.1
+
+# all_nulls_query_distance
+query R
+with data as (
+  select null::int as f, null::int as b
+  union all
+  select null::int as f, null::int as b
+)
+select distance(f, b)
+from data
+----
+0
```

Comment:
Should here be NULL? If so, I will modify the initial value of `sum_of_squares` in accumulator states too.